.github/workflows/publish.yaml

```diff
@@ -0,0 +1,27 @@
+name: publish Node.js Package to gpr
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+  release:
+    types: [created]
```

Comment:
This will trigger it _both_ when GH release is created _and_ when tag is pushed, right? I think tag is enough.